openapi.yaml

```diff
@@ -17139,6 +17145,13 @@ components:
             enough for it to be considered unhealthy and taken out of rotation.
           example: true
           x-linode-cli-display: 6
+        proxy_protocol:
```

Comment:
@hzoppetti it accepts `"v1"`, `"v2"`, and `"none"`, so it is not nullable

Reply:
updated enum